app/src/main/res/layout/fragment_folder.xml

```diff
@@ -221,38 +221,6 @@
                     android:layout_marginStart="75dp"
                     android:textAppearance="@style/TextAppearance.AppCompat.Caption" />
 
-            </LinearLayout>
-
-            <!-- variableSizeBlocks -->
-            <android.support.v7.widget.SwitchCompat
-                android:id="@+id/variableSizeBlocks"
-                style="@style/Widget.Syncthing.TextView.Label.Details"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:checked="false"
-                android:drawableLeft="@drawable/ic_developer_board_black_24dp"
-                android:drawableStart="@drawable/ic_developer_board_black_24dp"
-                android:text="@string/folder_variable_size_blocks_caption" />
-
-            <TextView
-                android:id="@+id/variableSizeBlocksDescription"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginLeft="75dp"
-                android:layout_marginStart="75dp"
-                android:layout_marginTop="-20dp"
-                android:textAppearance="@style/TextAppearance.AppCompat.Caption"
-                android:text="@string/folder_variable_size_blocks_description"
-                android:focusable="false"/>
-
-            <LinearLayout
-                android:id="@+id/editIgnoresContainer"
```

Comment:
Why is this removed?

Reply:
Huh, good point.

Reply:
It seems it slipped in with the original PR. Need to figure out if its used.

Reply:
Let me check please when I'm back from my vacation. From what I recall this is an extra layout container that came from the fork but is useless in official as there isn't the in-app edit .stignore UI. So at a first glance it can be removed.